.github/workflows/auto-comment.yml

```diff
@@ -13,4 +13,5 @@ jobs:
       - name: Process Label Action
```

Comment:
Should the hyphen in line 13 be removed? Then, should a hyphen be added to line 14?